nifi-commons/nifi-web-client-api/src/main/java/org/apache/nifi/web/client/api/HttpRequestBodySpec.java

```diff
@@ -31,4 +31,14 @@ public interface HttpRequestBodySpec extends HttpRequestHeadersSpec {
      * @return HTTP Request Headers Specification builder
      */
     HttpRequestHeadersSpec body(InputStream inputStream, OptionalLong contentLength);
+
+    /**
+     * Set Request Body as provided string
```

Comment:
Recommend indicating that the request body will be encoded as UTF-8.
```suggestion
     * Set Request Body as provided string encoded as UTF-8.
```